sw/otbn/code-snippets/README.md

```diff
@@ -1,15 +1,17 @@
-# OTBN code snippet collection
+---
+title: "OTBN Code Snippets"
+---
 
 This directory contains some code snippets that give examples of how
 to do various tasks in OTBN code.
 
-  - `modexp.s`: An example of how to do modular exponentiation.
-  - `pseudo-ops.s`: An example of the pseudo-operations supported by the OTBN ISA.
-  - `mul256.s`: An example of a 256x256 bit multiply using the MULQACC
+-   `modexp.s`: An example of how to do modular exponentiation.
+-   `pseudo-ops.s`: An example of the pseudo-operations supported by the OTBN ISA.
+-   `mul256.s`: An example of a 256x256 bit multiply using the MULQACC
     instruction.
-  - `mul384.s`: An example of a 384x384 bit multiply using the MULQACC
+-   `mul384.s`: An example of a 384x384 bit multiply using the MULQACC
     instruction.
-  - `barrett384.s`: An example of a modular multiplication kernel based on
+-   `barrett384.s`: An example of a modular multiplication kernel based on
     Barrett reduction.
```

Comment:
Heh, that's a neat way to avoid a big diff! But maybe it's cleaner to squash the resultant double space? Otherwise this markup is going to look a bit weird for the next person.

Reply:
Done.